### DISCUSSION TITLE
Search Button instead of Instant Search

### DISCUSSION THREAD

**Discussion by ngfw** [filamentphp/filament#8453] [Help]
Hello,

I'm working with FilamentPHP v2 and I'd like to know if there's a method or configuration option available to disable the Instant Search feature on a table and instead implement a manual Search button. Could you please provide guidance or instructions on how to achieve this?

<img width="451" alt="Screen Shot 2023-09-13 at 3 15 01 PM" src="https://github.com/filamentphp/filament/assets/203863/e9a0077a-e833-48ea-8c34-85ddeceab2ca">

**Comment by leandrocfe** [comment]
There isn't I think
